academy/2-cosmos-concepts/1-architecture.md

```diff
@@ -25,7 +25,7 @@ In this section, you will deepen your understanding of the application architect
 
 ## What are Tendermint and CometBFT?
 
-Created in 2014, [Tendermint](https://tendermint.com/) accelerates the development of distinct blockchains with a ready-made networking and consensus solution, so developers do not have to recreate these features for each new case. You may already use Tendermint without being aware of it, as other blockchains like [Hyperledger Burrow](https://hyperledger.github.io/burrow/#/) and the [Binance Chain](https://www.binance.org/en/smartChain) use Tendermint.
+Created in 2014, [Tendermint](https://tendermint.com/) accelerates the development of distinct blockchains with a ready-made networking and consensus solution, so developers do not have to recreate these features for each new case. You may already use Tendermint without being aware of it, as other blockchains like [Hyperledger Burrow](https://hyperledger.github.io/burrow/#/) (now discontinued) and the [Binance Chain](https://www.binance.org/en/smartChain) use Tendermint.
```

Comment:
@murataniloener Do you have an idea with what to replace this discontinued product?

Reply:
@samricotta Any chain project that would be worth mentioning?